test/intersections.jl

```diff
@@ -1188,7 +1189,7 @@
     quad = Quadrangle(cart(0, 0), cart(0.1, 0.0), cart(0.1, 0.1), cart(0.0, 0.1))
     poly = PolyArea(cart(0, 0), cart(2, 0), cart(1, 1), cart(1, 0.5))
     @test intersection(quad, poly) |> type == Intersecting
-    @test all(vertices(quad ∩ poly) .≈ [cart(0, 0), cart(0.1, 0), cart(0.1, 0.05)])
+    @test all(vertices(quad ∩ poly) .≈ [cart(0.1, 0.05), cart(0, 0), cart(0.1, 0)])
```

Comment:
Please make sure that the result is returned in CCW orientation.

Reply:
As far as I can tell, the method returns the correct orientations, as long as the input polygons are correctly oriented. The changes here are just changing the first vertex from which the new polygon starts. This is to be expected because the Weiler-Atherton searches for the first "Entering" vertex, while Sutherland-Hodgman starts from the first vertex of the to be clipped polygon. Algorithm does not try to ensure correct orientations for the input or output polygon rings, as in the cases for the clipping polygons with the holes, it relies on the output following the orientation of the input, because the algorithm is executed on outer and inner rings separately. In all my tests it looks to be working correctly.